nucypher/blockchain/eth/interfaces.py

```diff
@@ -631,7 +645,8 @@ def send_transaction(self,
 
         receipt = self.sign_and_broadcast_transaction(transaction_dict=transaction,
                                                       transaction_name=transaction_name,
-                                                      confirmations=confirmations)
+                                                      confirmations=confirmations,
+                                                      fire_and_forget=fire_and_forget)
         return receipt
```

Comment:
We should use `txhash_or_receipt` here instead of receipt

Reply:
:heavy_check_mark: